client/pages/login.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <div>
+    <LoginForm />
+  </div>
```

Comment:
Unnecessary use of `div`.
```suggestion
  <LoginForm />
```